test/metabase/models/setting_test.clj

```diff
@@ -182,6 +183,17 @@
         setting)))
 
 
+(defsetting test-i18n-setting
```

Comment:
`^:private`? :D

Reply:
👍 - didn't know defsetting support that. I'll mark the other defsettings in the file as private as well

Reply:
yeah it's just defining a function after all